src/Lumi/Styles.purs

```diff
@@ -1,40 +1,31 @@
 module Lumi.Styles
-  ( styleModifier
-  , styleModifier_
+  ( StyleModifier
+  , style
+  , style_
   , toCSS
   , module Emotion
   ) where
 
 import Prelude
-import Lumi.Components (PropsModifier, LumiProps, propsModifier)
+import Lumi.Components (PropsModifier, propsModifier)
 import Lumi.Styles.Theme (LumiTheme)
-import React.Basic.Emotion hiding (element) as Emotion
+import React.Basic.Emotion hiding (element, style) as Emotion
 
--- | Lift a themed set of styles into a `PropsModifier` for composition with other modifiers.
--- |
--- | Note: A style modifier should generally leave the `props` type unconstrained and take
--- |   configuration as regular arguments instead. Adding constraints to `props` makes it
--- |   difficult to compose style and prop modifiers together across different components,
--- |   where the same field name could mean different things.
-styleModifier :: forall props. (LumiTheme -> Emotion.Style) -> PropsModifier props
-styleModifier f = propsModifier \props -> props { css = f <> props.css }
+type StyleModifier = forall props. PropsModifier props
 
--- | Lift a static set of styles into a `PropsModifier` for composition with other modifiers.
--- |
--- | Note: A style modifier should generally leave the `props` type unconstrained and take
--- |   configuration as regular arguments instead. Adding constraints to `props` makes it
--- |   difficult to compose style and prop modifiers together across different components,
--- |   where the same field name could mean different things.
-styleModifier_ :: forall props. Emotion.Style -> PropsModifier props
-styleModifier_ = styleModifier <<< const
+-- | Lift a themed set of styles into a `StyleModifier` for composition with other modifiers.
+style :: (LumiTheme -> Emotion.Style) -> StyleModifier
+style f = propsModifier \props -> props { css = f <> props.css }
```

Comment:
@hdgarrood @spicydonuts I was actually thinking of also changing the signature of this function and of `style_` to take in an `Array` of styles instead:
```
style :: Array (LumiTheme -> Emotion.Style) -> StyleModifier

style_ :: Array Emotion.Style -> StyleModifier
```
This should make it easier (and cleaner) to apply multiple emotion styles conditionally. What do you think?

Reply:
I usually use `<>` for this, but I don't mind either way. We end up with parens around `style`'s arg 99% of the time anyway.

Reply:
Yeah, that's what I saw. Using brackets instead of parens at least removes the need for applying `merge` or `fold` anyway.

Reply:
Yeah, sounds good to me

Reply:
I'll actually just leave the change set to a minimum. If we ever really need that utility we can always just add new functions `styles` and `styles_`.